catalog/nsili/catalog-nsili-sourcestoquery-ui/src/main/webapp/js/application.js

```diff
@@ -12,24 +12,31 @@
 /*global define, require */
 
 // #Main Application
-define([
-    'backbone.marionette',
-    'bootstrap'
-], function (Marionette) {
+define(['underscore', 'backbone', 'marionette', 'icanhaz'], function(
+  _,
+  Backbone,
+  Marionette,
+  ich
+) {
     'use strict';
 
-    var Application = {};
+  var Application = {};
 
-    Application.App = new Marionette.Application();
+  // This was moved from the main.js file into here.
+  // Since this modules has ui components, and it gets loaded before main.js, we need to init the renderer here for now until we sort this out.
```

Comment:
Does this need a TODO and ticket number?

Reply:
I don't think there is any action here; we call the render in application instead of main.  The comment was taken from DDF